.meta/oss-pledge.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
is this file needs to under public/ directory so it fetchable?

Reply:
It's available via Github repo link 👍